dataland-frontend/tests/unit/services/DataStoreTest.spec.ts

```diff
@@ -19,14 +19,15 @@ describe("DataStore", () => {
             ],
             "type": "object",
             "properties": {
-                "Capex": {
-                    "$ref": "#/components/schemas/EuTaxonomyData"
+                "industrialSector": {
+                    "type": "string"
                 },
-                "Opex": {
-                    "$ref": "#/components/schemas/EuTaxonomyData"
+                "marketCap": {
+                    "type": "number"
                 },
-                "Revenue": {
-                    "$ref": "#/components/schemas/EuTaxonomyData"
+                "reportingDateOfMarketCap": {
```

Comment:
Dieses Objekt verstehe ich nicht ganz. Warum werden hier Company-spezifische Attribute wie market cap vermengt mit Report-spezifischen Attributen wie reporting obligation und warum werden teils Variablennamen verwendet und teils Klartextnamen?

Reply:
Die Werte hier haben keine Bedeutung und nur zum Testen.
allerdings finde ich guten Punkt und habe ich die Namen in diesem Test umbennant.